geometries/CMS_Phase2/Pixel/Pixel_V7/FPIX1_7_1_0.cfg

```diff
@@ -0,0 +1,454 @@
+  Endcap FPIX_1 {    
+    phiSegments 4
+    etaCut 10
+    zError 70
+    
+    trackingTags pixel,tracker
+
+    @include-std CMS_Phase2/Pixel/Materials/disk_FPIX
+    @include-std CMS_Phase2/Pixel/Conversions/On_flange/flange_FPIX
+    @include-std CMS_Phase2/Pixel/Conversions/On_services_cylinder/stations_serving_TFPX_615
+    @include-std CMS_Phase2/Pixel/Materials/Routing/routing_around_TFPX
+    
+    moduleShape rectangular
+    alignEdges true 
+    numDisks 8
+    bigDelta 0
+    outerRadius 160
+    numRings 4
+    barrelGap 33.325
+    maxZ 1300
+    bigParity 1
+    smallParity -1
+    zRotation 1.570796327
+    Ring 1 {
+      @include-std CMS_Phase2/Pixel/ModuleTypes/pixel_1x2_25x100_wide_3D
+      @include-std CMS_Phase2/Pixel/Materials/module_FPIX_R1_1x2_2500_3D
+      @include-std CMS_Phase2/Pixel/Resolutions/25x100
+      numModules 20
+      ringOuterRadius 74.45 
+      phiShift -0.008726646259972
+      smallDelta 9
+      Module 0 {
+       twistAng -0.148353
+       rhoCentre 53.31
+      }
+      Module 1 {
+       twistAng -0.1153663
+       rhoCentre 53.08
+      }
+      Module 2 {
+       twistAng -0.0349066
+       rhoCentre 52.76
```

Comment:
I would have expected 3 variables here:
- Xshift         // translation of sensor center of reference in X on the dee plane
- Yshift         // translation of sensor center of reference in Y on the dee plane
- yawAngle  // rotation of the sensor around its center, on the dee surface
Does that mean that all sensors are equidistant in phi?

In any case, maybe does not harm to have the 3 degrees of liberty, so that any further adjustment can be integrated?

Please note that the precision can be more limited using angles than distances at the CAD -> cfg files interface.
For yawAngle we have no choice, but for the sensor translation, a translation can be good.

Reply:
The sensors aren't all equidistant in phi: there are 2 places around the ring where there is additional space, the exact placement is determined in Ring.cc based on the number of modules and the phiShift() - of course it could also have been passed here on a module-by-module basis, can see benefits to either :-) 

In any case, the x and y shifts are still determined as `r cos(phi)` and `r sin(phi)` (as for modules in a regular ring) - IMO it's less error-prone to let the code determine those shifts than to calculate them by hand beforehand.